smallfile_wrapper/Dockerfile

```diff
@@ -1,10 +1,14 @@
 FROM centos/tools
 
-USER root
-WORKDIR /root/
 RUN yum install git epel-release -y
-RUN yum install python2-elasticsearch python2-numpy python2-configparser python2-redis -y
+RUN yum install python2-pip python2-numpy scipy python2-configparser python2-redis -y
+RUN pip install --upgrade pip
+RUN pip install elasticsearch statistics
 RUN git clone https://github.com/distributed-system-analysis/smallfile
-RUN mv smallfile smallfile-master
-RUN mkdir -p /opt/snafu/
-COPY . /opt/snafu/
+RUN mv smallfile /opt/
+RUN ln -sv /opt/smallfile/smallfile_cli.py /usr/local/bin/
+RUN ln -sv /opt/smallfile/smallfile_rsptimes_stats.py /usr/local/bin/
+RUN git clone https://github.com/bengland2/snafu
```

Comment:
why are using the fork here?

Reply:
oops will fix, I meant to just use that for development purposes not in the actual commit.